run_e3sm.template.sh

```diff
@@ -46,8 +43,8 @@ readonly GET_REFCASE=TRUE
 #readonly RUN_REFDATE=""   # same as MODEL_START_DATE for 'branch', can be different for 'hybrid'
 
 # Set paths
-readonly CODE_ROOT="${HOME}/E3SMv3/code/${CHECKOUT}"
-readonly CASE_ROOT="/pscratch/sd/r/${USER}/e3sm-scratch/${CASE_NAME}"
+readonly CASE_ROOT="${PSCRATCH}/e3sm-scratch/${MACHINE}/${CASE_NAME}"
```

Comment:
I think the source code should be on HOME.  Its lots of small files.

Reply:
Is anyone using this convention for their CASE_ROOT? I've never used the old tradition of 'e3sm-scratch'

It would be more elegant if CASE_ROOT had a similar structure as CODE_ROOT, for example:

CASE_ROOT="${PSCRATCH}/E3SMv3/${CASE_NAME}"